onnxruntime/python/tools/transformers/models/whisper/benchmark.py

```diff
@@ -411,6 +411,8 @@ def handle_output(output):
         logger.info(f"Generated token length: {len(actual_output)} tokens")
         transcription = args.processor.batch_decode(ort_outputs[0], skip_special_tokens=True)[0]
         logger.info(f"Transcription: {transcription}")
```

Comment:
If you need to print to stdout for comparison, is this line needed then? We should not print the transcription twice.